views/AddInventory.html

```diff
@@ -30,8 +35,11 @@
 
     <div class="row form-group">
         <div class="col-sm-4">
-            <label for="location_input">Location</label>
-            <input id="location_input" class="form-control" title="The location this inventory will be stored at."></input>
+            <form>
+                        Location
+                        <input type="text" id="location_input" name="location_input" value="A-1"
```

Comment:
I don't think we want a default value for the location. Ideally it would be the last location that was used for that product, but that sounds like too much work. Probably having none or just the first item in the list (if exists) is best.